### PR TITLE
Ros1 update readme

### PR DESCRIPTION
**Pull request by Kotochleb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,69 @@
 # panther_ros
 
-Packages composing basic funcinalities of the Husarion Panther robot.
\ No newline at end of file
+Packages composing basic functionalities of the Husarion Panther robot.
+API for each node can be found in each package folder.
+
+## Build and run on hardware
+
+In order to build the Panther for hardware setup on internal computer, use following commands.
```
Comment (KmakD) [inline README.md]:
```suggestion
In order to build the Panther ROS interface for hardware setup on internal computer, use following commands:
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,69 @@
...
+## Build and run simulation
+
+To build your system run:
+``` bash
+export HUSARION_ROS_BUILD_TYPE=simulation
+
+pip3 install rosdep vcstool
+
+git clone https://github.com/husarion/panther_ros.git src/panther_ros
+vcs import src < src/panther_ros/panther/panther.repos
```
Comment (KmakD) [inline README.md:65]: Don't we want to remove unnecessary packages? The fastest way will be something like this I guess:
```
mv panther_ros/panther_description/ src/panther_description
mv panther_ros/panther_gazebo/ src/panther_gazebo
rm -rf panther_ros
```

**Review by KmakD — CHANGES_REQUESTED** [review]

**Comment by pkowalsk1** [comment]
I think we should include information about our official way to launch our ROS packages with docker configuration. Launching nodes on host should work, but it's not a recommended solution.

**Review by pkowalsk1 — CHANGES_REQUESTED** [review]
I think we should include very clear information about our official way to launch our ROS packages with docker configuration. Launching nodes on host should work, but it's not a recommended solution.

File: README.md (hunk 2 above)
Comment (Kotochleb) [reply README.md:65]: Done

File: README.md [hunk 3]
```diff
@@ -1,3 +1,103 @@
...
+In order to download it, run:
+``` bash
+docker pull husarion/panther
+```
+
+Example **compose.yaml** can be found on [husarion/panther-docker](https://github.com/husarion/panther-docker/) in **demo** section.
+
+## Build and run on hardware
+
+To build hardware interface packages running on for Panther robot's internal computer, use the following commands:
```
Comment (KmakD) [inline README.md]:
```suggestion
To build hardware interface packages running on the Panther robot's internal computer, use the following commands:
```

**Review by KmakD — CHANGES_REQUESTED** [review]

**Review by KmakD — APPROVED** [review]

**Review by pkowalsk1 — APPROVED** [review]

**Review by KmakD — APPROVED** [review]

**Pull request merged**